### ISSUE TITLE
[Waves Screen] Energy Balance checkbox is missing accessible name

### ISSUE THREAD

**Issue by arouinfar** [phetsims/greenhouse-effect#365]
The Energy Balance checkbox seems to be missing an accessible name. It shows up as an unlabelled checkbox in the a11y view on main.

Here's what the end of the Play Area looks like:
![Screen Shot 2023-10-11 at 4 27 57 PM](https://github.com/phetsims/greenhouse-effect/assets/8419308/a9ec98af-a1c2-42c8-9de4-5da50635390c)

Assigning to @jbphet but deferring because development is currently paused.

**Comment by jbphet** [comment]
The accessible name has been added, closing.
